### PR TITLE
Add library-fees test for incorrect date comparisons

### LINKED ISSUE #1096: Library Fees Tests 25 & 26 are one day off
https://github.com/exercism/elixir/blob/c82ea9b2f35b083f2463a04494ee4dc2a47a6185/exercises/concept/library-fees/test/library_fees_test.exs#L150-L160

All tests are successful, except the last two, which are exactly one day off.
Changing the function to accommodate the offset makes other tests fail.
Even previous solutions now seem to fail.

What am I missing here?

### PR DESCRIPTION
**Pull request by angelikatyborska**
Resolves #1096

### CODE REVIEW

**Comment by jiegillet** [comment]
@glennj  and @jnnks, would you care to review this PR? I haven't look closely at that issue yet.

**Comment by jnnks** [comment]
Looks good to me, thank you @angelikatyborska :+1:

File: exercises/concept/library-fees/test/library_fees_test.exs [hunk 1]
```diff
@@ -76,6 +76,12 @@ defmodule LibraryFeesTest do
       result = LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-21 16:00:00Z])
       assert result == 9
     end
+
+    @tag task_id: 4
+    test "a new day starts at midnight" do
+      result = LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-13 00:00:00Z])
+      assert result == 1
+    end
```
Comment (glennj) [inline exercises/concept/library-fees/test/library_fees_test.exs:84]: What about this, to really drive home the point.
```elixir
    test "a new day starts at midnight" do
      result = [
        LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-13 23:59:59Z]),
        LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-14 00:00:00Z]),
      ]
      assert result == [1, 2]
    end
```

Comment (angelikatyborska) [reply exercises/concept/library-fees/test/library_fees_test.exs:84]: sure, but as separate assertions

Comment (glennj) [reply exercises/concept/library-fees/test/library_fees_test.exs:87]: Looks good to me!

**Review by glennj — APPROVED** [review]

**Comment by angelikatyborska** [comment]
@jiegillet your approval is still needed to merge 😅

**Review by jiegillet — APPROVED** [review]

**Pull request merged**